docs/deployment/kyuubi_on_kubernetes.md

```diff
@@ -90,7 +90,7 @@ See more related details in [Using RBAC Authorization](https://kubernetes.io/doc
 
 ## Config
 
-You can configure Kyuubi the old-fashioned way by placing kyuubi-default.conf inside the image. Kyuubi do not recommend using this way on Kubernetes.
+You can configure Kyuubi the old-fashioned way by placing kyuubi-default.conf inside the image. Kyuubi does not recommend using this way on Kubernetes.
```

Comment:
There is another issue in this statement

```suggestion
You can configure Kyuubi the old-fashioned way by placing `kyuubi-defaults.conf` inside the image. Kyuubi does not recommend using this way on Kubernetes.
```

Reply:
Thanks @pan3793 - I've updated the docs with your correction.